java/runtime/src/main/java/org/ray/runtime/actor/NativeRayActor.java

```diff
@@ -73,29 +70,45 @@ public boolean isDirectCallActor() {
 
   @Override
   public void writeExternal(ObjectOutput out) throws IOException {
-    out.writeObject(nativeSerialize(nativeCoreWorkerPointer, actorId));
-    out.writeObject(language);
+    out.writeObject(toBytes());
   }
 
   @Override
   public void readExternal(ObjectInput in) throws IOException, ClassNotFoundException {
+    nativeCoreWorkerPointer = getNativeCoreWorkerPointer();
+    actorId = nativeDeserialize(nativeCoreWorkerPointer, (byte[]) in.readObject());
+  }
+
+  public byte[] toBytes() {
```

Comment:
document: `Serialize this actor handle to bytes.`

Reply:
OK